### PR TITLE
Feature/locator impl upgrade script

### PR DESCRIPTION
**Pull request by bulbozaur**

### CODE REVIEW

File: scripts/upgrade_lido_locator.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    prompt_bool,
+    get_deployer_account,
+    lido_dao_template_address,
+    lido_dao_lido_locator_implementation,
+    contracts,
+)
+
+
+def main():
+
```
Comment (TheDZhon) [inline scripts/upgrade_lido_locator.py:16]: Please consider increasing the max fee from `100 gwei` to `300 gwei`.
The network is busy this week.

File: scripts/upgrade_lido_locator.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        resume = prompt_bool()
+
+    if not resume:
+        print("Exit without running.")
+        return False
+
+    interface.OssifiableProxy(contracts.lido_locator).proxy__upgradeTo(
+        lido_dao_lido_locator_implementation, get_tx_params(deployer)
+    )
+
```
Comment (TheDZhon) [inline scripts/upgrade_lido_locator.py:37]: I would add the additional assert here that the implementation is upgraded :fearful:

File: scripts/upgrade_lido_locator.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+)
+
+
+def main():
+
+    deployer = get_deployer_account() if "DEPLOYER" in os.environ else accounts.at("0x2A78076BF797dAC2D25c9568F79b61aFE565B88C", force=True)
+
+    assert interface.OssifiableProxy(contracts.lido_locator).proxy__getAdmin() == deployer
+
+    print(f"=== Upgrade lido locator implementation to {lido_dao_lido_locator_implementation} ===")
```
Comment (TheDZhon) [inline scripts/upgrade_lido_locator.py:21]: Please add a warning for the live network

**Review by TheDZhon — APPROVED** [review]
:+1: 

Left some ideas for double-checking the outcome.

File: scripts/upgrade_lido_locator.py (hunk 2 above)
Comment (bulbozaur) [reply scripts/upgrade_lido_locator.py]: Splitted into 2 separate scripts with asserts

**Review by rkolpakov — APPROVED** [review]
lgtm

**Pull request merged**